MSY/Day1/문자열 내 p와 y의 개수.js

```diff
@@ -0,0 +1,18 @@
+function solution(s){
+    let pCount = 0;
+    let yCount = 0;
+
+    for (let i = 0; i < s.length; i++) {
+        if(s[i]==='p' || s[i]==='P') {
+            pCount++;
+        }
+        if(s[i]==='y' || s[i]==='Y') {
+            yCount++;
+        }
+    }
+    if (pCount===yCount) {
+        return true;
+    } else {
+        return false;
+    }
```

Comment:
```suggestion
    for (let i = 0; i < s.length; i++) {
        if(s[i] === 'p' || s[i] === 'P') {
            pCount++;
        }
        if(s[i] === 'y' || s[i] === 'Y') {
            yCount++;
        }
    }
    if (pCount === yCount) {
        return true;
    } else {
        return false;
    }
```

전체적으로 `pCount`, `yCount`처럼 변수명을 잘 작성하신것 같습니다! 스페이스로 구분을 주시면 가독성이 더욱 올라갈것 같아요 👍

Reply:
넵! 감사합니다!